lib/synapse/service_watcher/multi/resolver/README.md

```diff
@@ -0,0 +1,56 @@
+# Resolvers
+
+A resolver decides how to combine, or resolve, multiple service watchers into a
+single result. That is, it operates as a `reducer` function to allow more than
+one service watchers to appear, and act, as a single watcher.
+
+The stub methods listed below should be overridden by any children classes.
+If any additional methods are overridden (such as `initialize`), be sure to
+call `super` first.
+
+```ruby
+require "synapse/service_watcher/multi/resolver/base"
+
+class Synapse::ServiceWatcher::MultiWatcher::Resolver
+   class MyResolver < BaseResolver
+      def start
+	     # start resolver
+	  end
+
+	  def stop
+	     # stop resolver
+	  end
+
+	  def backends
+	     # return a single list of backends
+	  end
+
+	  def ping?
+	     # return whether or not the watchers are healthy
+	  end
```

Comment:
find a better name of the method signature and make it more clear in the comments